base/etc/condor-ce/config.d/70-container.conf

```diff
@@ -51,3 +51,8 @@ AUTH_SSL_CLIENT_KEYFILE = /etc/grid-security/hostkey.pem
 # Tell clients to contact the CE on the hostname used to
 # advertise to other daemons
 TCP_FORWARDING_HOST = $(NETWORK_HOSTNAME)
+
+# FIXME: Remove this when after it's been upstreamed (HTCONDOR-1106)
+# Ensure that the CE only listens on 9619
+COLLECTOR_PORT = $(PORT)
+SHARED_PORT_PORT = $(PORT)
```

Comment:
Please configure your editor not to omit the end-of-line from the final line in the file.

Of course, this doesn't explicitly set the ports to `9619`, it just sets them to match the configured `PORT`.

Otherwise, assuming `PORT` comes from [here](https://github.com/htcondor/htcondor-ce/blob/master/config/condor_config#L14), I guess this is straightforward enough.

Reply:
Fixed and updated the commit/PR message to boot